res/xmaskye_jr.xml

```diff
@@ -1,66 +1,66 @@
-<?xml version="1.0" encoding="ISO-8859-1"?>
-<!---xyeskin, specify an image file (which must be in the same folder and be of png format) and the size of the grid-->
-<!---You can also customize the font for some reason.-->
-<xyeskin
-sprites="XMASKye.png"
-luminosity="XMASKye_luminosity.png"
-icon='kye_icon.png'
-size='16'
-fontfile="DejaVuSans.ttf"
-boldfontfile="DejaVuSans-Bold.ttf"
-
-xyedirections ='no'
-truetypesize='11'
-
->
-
-
-<title>Christmas Kye Jr.</title>
-<description>This skin reproduces the look and feel of Christmas Kye Jr. Most sprites are from that game, others were adapted by Justin de Lucia. This variation of the skin uses the brighter walls from Kye Jr.</description>
-
-<levelmenu>
-    <info     red='220' green='220' blue='255' />
-    <selected      red='0' green='0' blue='128' />
-    <selectederror red='255' green='0' blue='0' />
-    <menu          red='255' green='255' blue='255' />
-    <menutext      red='0' green='0' blue='0' />
-    <selectedtext  red='255' green='255' blue='255' />
-    <infotext      red='0' green='0' blue='0' />
-</levelmenu>
-
-<!-- wall default color: -->
-<color type ="WALL" red="0" green="191" blue="191" />
-<!-- wall color in the sprite: -->
-<color type ="WALL_SPRITE" red="255" green="255" blue="255" />
-
-
-<!-- Yellow blocks: -->
-<color type="B" bc="Y" red='255' green='255' blue='150'/>
-<color type="F" bc="Y" red='0' green='191' blue='191'/>
-
-<!-- red blocks: -->
-<color type="B" bc="R" red='255' green='200' blue='200'/>
-<color type="F" bc="R" red='255' green='150' blue='150'/>
-
-<!-- Blue blocks: -->
-<color type="B" bc="B" red='150' green='255' blue='255'/>
-<color type="F" bc="B" red='0' green='191' blue='191'/>
-
-<!-- Green blocks: -->
-<color type="B" bc="G" red='180' green='230' blue='180'/>
-<color type="F" bc="G" red='0' green='191' blue='191'/>
-
-
-<!-- What color to use for one-way doors -->
-<color type='ONEWAYDOOR' red='255' green='0' blue='0' />
-
-<!-- What color to use for force arrows -->
-<color type='FORCEARROW' red='100' green='200' blue='200' />
-
-<!-- What color to use for earth / soft-blocks -->
-<color type='EARTH' red='255' green='255' blue='255' />
-
-<!-- What color to use for the background/floor -->
-<color type='FLOOR' red='255' green='255' blue='255' />
-
-</xyeskin>
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
Speaking of clean git history, there doesn't seem liek there's a difference between these files. Possibly just a line breaks thing. For that reason maybe there is no reason to include them in the PR

Reply:
It's again the permission as executable on an XML file that's been removed. I think the line breaks have been altered by Git itself that normally acts to remove the Windows line breaks on the repo (for file types where this makes sense) and re-established them or not when cloning depending on whether your are on a Windows machine or a Linux/macOS one.
I'd suggest to keep the change as it's a normal behavior for cross-platform projects.